src/main/java/gregtech/api/recipes/machines/RecipeMapAssemblyLine.java

```diff
@@ -70,4 +68,14 @@ protected void addInventorySlotGroup(ModularUI.Builder builder, @Nonnull IItemHa
             addSlot(builder, startInputsX + 18 * 4, 1, 0/*18*/, itemHandler, fluidHandler, invertFluids, true); // Output Slot - 18 for data slot
         }
     }
+
+    @Override
+    public void setMaxInputs(int maxInputs) {
+        throw new UnsupportedOperationException("Cannot change output amount for Assembly Line.");
+    }
+
+    @Override
+    public void setMaxFluidInputs(int maxFluidInputs) {
```

Comment:
Should this be overriding the fluid outputs? Kinda confused because the error says outputs, method says inputs

Reply:
Was a copy-paste oversight. It should be inputs.